src/compiler/types/tests/ComponentCompilerMeta.stub.ts

```diff
@@ -9,17 +9,85 @@ import * as d from '@stencil/core/declarations';
  */
 export const stubComponentCompilerMeta = (
   overrides: Partial<d.ComponentCompilerMeta> = {}
-): d.ComponentCompilerMeta => {
-  // TODO(STENCIL-378): Continue to build out default stub, remove the type assertion on `default`
-  const defaults: d.ComponentCompilerMeta = {
-    isCollectionDependency: false,
-    events: [],
-    methods: [],
-    properties: [],
-    sourceFilePath: '/some/stubbed/path/my-component.tsx',
-    tagName: 'stub-cmp',
-    virtualProperties: [],
-  } as d.ComponentCompilerMeta;
-
-  return { ...defaults, ...overrides };
-};
+): d.ComponentCompilerMeta => ({
+  assetsDirs: [],
+  componentClassName: '',
+  dependencies: [],
+  dependents: [],
+  directDependencies: [],
+  directDependents: [],
+  docs: { text: 'docs', tags: [] },
+  elementRef: '',
+  encapsulation: 'none',
+  events: [],
+  excludeFromCollection: false,
+  hasAttribute: false,
+  hasAttributeChangedCallbackFn: false,
+  hasComponentDidLoadFn: false,
+  hasComponentDidRenderFn: false,
+  hasComponentDidUnloadFn: false,
+  hasComponentDidUpdateFn: false,
+  hasComponentShouldUpdateFn: false,
+  hasComponentWillLoadFn: false,
+  hasComponentWillRenderFn: false,
+  hasComponentWillUpdateFn: false,
+  hasConnectedCallbackFn: false,
+  hasDisconnectedCallbackFn: false,
+  hasElement: false,
+  hasEvent: false,
+  hasLifecycle: false,
+  hasListener: false,
+  hasListenerTarget: false,
+  hasListenerTargetBody: false,
+  hasListenerTargetDocument: false,
+  hasListenerTargetParent: false,
+  hasListenerTargetWindow: false,
+  hasMember: false,
+  hasMethod: false,
+  hasMode: false,
+  hasProp: false,
+  hasPropBoolean: false,
+  hasPropMutable: false,
+  hasPropNumber: false,
+  hasPropString: false,
+  hasReflect: false,
+  hasRenderFn: false,
+  hasState: false,
+  hasStyle: false,
+  hasVdomAttribute: false,
+  hasVdomClass: false,
+  hasVdomFunctional: false,
+  hasVdomKey: false,
+  hasVdomListener: false,
+  hasVdomPropOrAttr: false,
+  hasVdomRef: false,
+  hasVdomRender: false,
+  hasVdomStyle: false,
+  hasVdomText: false,
+  hasVdomXlink: false,
+  hasWatchCallback: false,
+  htmlAttrNames: [],
+  htmlParts: [],
+  htmlTagNames: [],
+  internal: false,
+  isCollectionDependency: false,
+  isPlain: false,
+  isUpdateable: false,
+  jsFilePath: '/some/stubbed/path/my-other-component.js',
+  legacyConnect: [],
+  legacyContext: [],
+  listeners: [],
+  methods: [],
+  potentialCmpRefs: [],
+  properties: [],
+  shadowDelegatesFocus: false,
+  sourceFilePath: '/some/stubbed/path/my-component.tsx',
+  sourceMapPath: '/some/stubbed/path/my-other-component.map',
```

Comment:
Should the source map file line up with the source?

```suggestion
  sourceMapPath: '/some/stubbed/path/my-component.js.map',
```

Reply:
probably!